release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1445,17 +1443,17 @@ In the table, features are marked with the following statuses:
 |vSphere 6.7 Update 2 or earlier
 |GA
 |DEP
-|DEP
+|REM
 
 |Virtual hardware version 13
```

Comment:
Isn't Virtual hardware version 14 also deprecated as well?

Reply:
@sgaoshang I've updated the PR with some feedback from Ju. Can you take another look and confirm whether HW version 14 should also be added as deprecated? Thank you

Reply:
@bscott-rh Yes, I think so. According to CSI driver requirement, we should use HW version 15 from OCP 4.9, details see https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-vsphere-and-virtual-hardware-version-deprecations @duanwei33 pls correct me if I'm wrong, thanks.

Reply:
hmm, I remember we had discussion for HW version 14, seems we never announce we support 14, so we did not mention that in our deprecated part previously.  
I think it is safe to keep current status(?) Otherwides, we need to update all doc according to this topic.